flink-connectors/flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/dialect/JdbcDialect.java

```diff
@@ -57,6 +57,8 @@
 	 */
 	JdbcRowConverter getRowConverter(RowType rowType);
 
+	String getLimitStatement(long limit);
```

Comment:
Please add javadoc on the method, including the description for limit parameter, e.g. the value range.